sql/updates/world/3.3.5/2017_10_20_99_world_alchemy_spec.sql

```diff
@@ -0,0 +1,112 @@
+--
+SET @TEXTID := 11074;
+DELETE FROM `npc_text` WHERE `ID`=@TEXTID;
+INSERT INTO `npc_text` (`ID`, `text0_1`, `BroadcastTextID0`) VALUES
+(@TEXTID, "Forgetting your skill in Potion Mastery is not something to do lightly.$B$BAre you absolutely sure?", 21870);
+
+SET @MENUID := xxx; -- Need 3
```

Comment:
insert 21402 for now to avoid travis-ci failing (change later if you find a documented one)

Reply:
Done, thanks :)